book/validation.rst

```diff
@@ -639,27 +658,27 @@ this method must return ``true``:
 
     .. code-block:: php
 
-        // src/Acme/BlogBundle/Entity/Author.php
+        // src/AppBundle/Entity/Author.php
 
         // ...
         use Symfony\Component\Validator\Mapping\ClassMetadata;
-        use Symfony\Component\Validator\Constraints\True;
+        use Symfony\Component\Validator\Constraints as Assert;
 
         class Author
         {
             public static function loadValidatorMetadata(ClassMetadata $metadata)
             {
-                $metadata->addGetterConstraint('passwordLegal', new True(array(
+                $metadata->addGetterConstraint('passwordLegal', new Assert\True(array(
                     'message' => 'The password cannot match your first name',
                 )));
             }
         }
 
-Now, create the ``isPasswordLegal()`` method, and include the logic you need::
+Now, create the ``isPasswordLegal()`` method and include the logic you need::
 
     public function isPasswordLegal()
     {
-        return $this->firstName != $this->password;
+        return $this->firstName !== $this->password;
```

Comment:
I know this is a standard, but I still dislike this in many cases. Certainly, it makes no difference here. But technically, if I set `$author->firstName = 12345` and then the user types in the string `'12345'` when filling out a form, that *should* fail validation. Both will ultimately be saved as strings in the DB (even though I was being "silly" and setting it as an integer in my controller). So, they really are "equal". I just want us to be careful with these - I think always using `==` OR `===` can cause WTF moments - sometimes one is more appropriate than the other.